rewatch/src/build/clean.rs

```diff
@@ -329,6 +345,9 @@ pub fn cleanup_after_build(build_state: &BuildState) {
 pub fn clean(path: &Path, show_progress: bool, snapshot_output: bool, build_dev_deps: bool) -> Result<()> {
     let project_root = helpers::get_abs_path(path);
     let workspace_root = helpers::get_workspace_root(&project_root);
+    let workspace_config_name = &workspace_root
+        .as_ref()
+        .and_then(|wr| packages::read_package_name(wr).ok());
     let packages = packages::make(
```

Comment:
If workspace is Some and the parent/child link is there, all packages will be loaded. I think in general this is more correct.